staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -117,6 +120,64 @@ type MetricSpec struct {
 	External *ExternalMetricSource `json:"external,omitempty" protobuf:"bytes,5,opt,name=external"`
 }
 
+// HorizontalPodAutoscalerBehavior configures a scaling behavior for Up and Down direction
+// (scaleUp and scaleDown fields respectively)
+type HorizontalPodAutoscalerBehavior struct {
+	// constraint value for scaling Up
+	// +optional
+	ScaleUp *HPAScalingRules `json:"scaleUp,omitempty" protobuf:"bytes,1,opt,name=scaleUp"`
+	// constraint value for scaling Down
+	// +optional
+	ScaleDown *HPAScalingRules `json:"scaleDown,omitempty" protobuf:"bytes,2,opt,name=scaleDown"`
+}
+
+// ScalingPolicySelect is used to specify which policy should be used while scaling in a certain direction
+type ScalingPolicySelect string
+
+const (
+	// MaxPolicySelect selects the policy with the highest possible change
+	MaxPolicySelect ScalingPolicySelect = "max"
+	// MinPolicySelect selects the policy with the lowest possible change
+	MinPolicySelect ScalingPolicySelect = "min"
+	// DisabledPolicySelect disables the scaling in this direction
+	DisabledPolicySelect ScalingPolicySelect = "disabled"
+)
+
+// HPAScalingRules configures the scaling policy and the policy selector
+type HPAScalingRules struct {
+	// StabilizationWindowSeconds is the number of seconds for which past recommendations should be considered while
+	// scaling up or scaling down
+	// +optional
+	StabilizationWindowSeconds *int32 `json:"stabilizationWindowSeconds" protobuf:"varint,3,opt,name=stabilizationWindowSeconds"`
+	// SelectPolicy is used to specify which policy should be used
```

Comment:
Default when not specified

Reply:
Fixed: https://github.com/kubernetes/kubernetes/pull/74525/commits/877d2ffeccb6d8fd1e47a12eab02de44b8bac07b